source/client.c

```diff
@@ -1923,6 +1923,15 @@ void mqtt_resubscribe_existing_topics(struct aws_mqtt_client_connection *connect
     uint16_t packet_id =
         mqtt_create_request(task_arg->connection, &s_resubscribe_send, task_arg, &s_resubscribe_complete, task_arg);
 
+    if (packet_id == 0) {
+        AWS_LOGF_DEBUG(
+            AWS_LS_MQTT_CLIENT,
+            "id=%p: Failed to send multi-topic resubscribe with error %s",
+            (void *)connection,
+            aws_error_name(aws_last_error()));
+        return;
```

Comment:
Other than logging it out, we're not dealing with this or reporting it to the user in a way they can react to this now-not-super-valid connection